README.md

```diff
@@ -98,21 +105,15 @@ If you are developing a ticket that deals with any sort of monitoring or asynchr
 BACK_RAWLS=true ./config/docker-rsync-local-rawls.sh
 ```
 
-**Important**: It is highly recommended that use your own Cloud SQL instance when running an instance of back Rawls by cloning the [dev Rawls DB](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/overview?project=broad-dsde-dev). See note below on database work.
-
-#### Developing Database Schema Changes
-
-If you are writing Liquibase migrations or doing database work, it is mandatory that you create your own Cloud SQL instance and develop against that. To point to your own CloudSQL instance, modify `/config/sqlproxy.env` accordingly.
-
+### Developing Database Schema Changes
 
+If you are writing Liquibase migrations or doing database work, set up a database clone by following the steps for [running Back Rawls](#additional-requirements-for-back-rawls).
 
 ## Developer quick links:
 * Swagger UI: https://rawls.dsde-dev.broadinstitute.org
 
 ## Build Rawls docker image
 
-Note: this may use more than 4.5 GB of Docker memory. If your Docker is not configured with enough memory (Docker for Mac defaults to 2GB), you may see a cryptic error messages saying `Killed`.
```

Comment:
Is this just never an issue with docker set to 8+ gb?

Reply:
I overhauled the Docker build process a couple months ago and ran dozens of builds that worked fine with 8 GB.

I also think that "you may see a cryptic error message" is not that helpful without specifics, there are *lots* of ways to get cryptic errors with Docker.

Reply:
lol fair, there is more than one way to be cryptic